app/jobs/api/issues_label_issue_job.rb

```diff
@@ -1,11 +1,18 @@
 module Api
   class IssuesLabelIssueJob < IssueEventJob
-    include IsPublishable
     timestamp ->(params) { params[:issue]['updated_at'] }
     action "issue_labeled"
+    cache_key ->(message) {
+      label_name = message[:payload][:label] ? message[:payload][:label]['name'] : ""
+
+      "#{message[:meta][:action]}.#{message[:meta][:user]["login"]}.#{message[:meta][:identifier]}.#{message[:meta][:timestamp]}.#{ label_name }"
```

Comment:
[Added TODOs to DRY these up](https://github.com/huboard/huboard-web/pull/255/commits/0cbb4a365fa5f690f5007c2e471b3a592069a3a5)